### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by beikov**
I'm still going to add some documentation but in a separate commit, so you could already review these changes, but don't merge it yet please.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -440,6 +452,43 @@ After that, it's easiest to just invoke `./serve-website.sh` which builds the do
 Now you should be able to select *Blaze-Persistence Checkstyle rules* in the dropdown of the CheckStyle window. +
 Click on *Check project* and checkstyle will run once for the whole project, then it should do some work incrementally.
 
+## Testing a JPA provider and DBMS combination
+
+By default, a Maven build `mvn clean install` will test against H2 and Hibernate 5.2 but you can activate different profiles to test other combinations.
+To test a specific combination, you need to activate at least 3 profiles
```
[the remark this answers is not in the document]
Comment (beikov) [reply README.md]: Unfortunately it's not possible. Defining a single profile will disable the auto activation

File: website/src/main/jbake/content/news/2018/blaze-persistence-1.2.0-Alpha4-release.adoc [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+* https://github.com/Blazebit/blaze-persistence/issues/361[*#361*, window="_blank"] Entity view attribute converter support
+* https://github.com/Blazebit/blaze-persistence/issues/414[*#414*, window="_blank"] Updatable Entity views
+* https://github.com/Blazebit/blaze-persistence/issues/419[*#419*, window="_blank"] Improved keyset pagination implementation
+* https://github.com/Blazebit/blaze-persistence/issues/430[*#430*, window="_blank"] Entity view equality now based on entity type rather than entity view type
+* https://github.com/Blazebit/blaze-persistence/issues/443[*#443*, window="_blank"] DML support for entity collections
+* https://github.com/Blazebit/blaze-persistence/issues/496[*#496*, window="_blank"] Entity View convenience methods +++<!-- PREVIEW-SUFFIX --><!-- </p></li></ul></div> --><!-- PREVIEW-END -->+++ for access by id
+* https://github.com/Blazebit/blaze-persistence/issues/428[*#428*, window="_blank"] New archetypes for Spring Boot, Spring Data, Java EE and DeltaSpike Data
+
+and lots of bugfixes, most notably
+
```
[the remark this answers is not in the document]
Comment (Mobe91) [reply website/src/main/jbake/content/news/2018/blaze-persistence-1.2.0-Alpha4-release.adoc]: I previously commented that the tenses should be kept consistent. The same applies here.

File: documentation/src/main/asciidoc/entity-view/manual/en_US/07_querying_and_pagination_api.adoc [hunk 3]
```diff
@@ -21,6 +21,9 @@ A `EntityViewSetting` essentially is configuration that can be applied to a `Cri
 
 Allowing the actual data consumer i.e. the UI to specify these aspects is essential for efficient and easy to maintain data retrieval.
 
+For the simple case, the lookup by id, there is also a convenience link:{entity_view_jdoc}/persistence/view/EntityViewSetting.html#create(java.lang.Class)[`EntityViewManager.find()`] method available
```
Comment (Mobe91) [inline documentation/src/main/asciidoc/entity-view/manual/en_US/07_querying_and_pagination_api.adoc]: For a simple lookup by id there is also...

File: documentation/src/main/asciidoc/entity-view/manual/en_US/07_querying_and_pagination_api.adoc [hunk 4]
```diff
@@ -21,6 +21,9 @@ A `EntityViewSetting` essentially is configuration that can be applied to a `Cri
...
+that allows you to skip some of the `CriteriaBuilder` ceremony and works analogous to how `EntityManager.find()` works, but just with entity views.
```
Comment (Mobe91) [inline documentation/src/main/asciidoc/entity-view/manual/en_US/07_querying_and_pagination_api.adoc]: ... and that works analogous to how `EntityManager.find()` works, but with entity views.

File: documentation/src/main/asciidoc/entity-view/manual/en_US/08_updatable_entity_views.adoc [hunk 5]
```diff
@@ -612,6 +609,10 @@ By default, only instances of the declared type i.e. the compile time attribute
 
 NOTE: This will change in the future by https://github.com/Blazebit/blaze-persistence/issues/450
 
+In case of immutable or non-updatable subview types the method link:{entity_view_jdoc}/persistence/view/EntityViewManager.html#getReference(java.lang.Class,%20java.lang.Object)[`EntityViewManager.getReference(Class viewType, Object id)`] might come handy.
```
Comment (Mobe91) [inline documentation/src/main/asciidoc/entity-view/manual/en_US/08_updatable_entity_views.adoc]: ... might come in handy.

File: documentation/src/main/asciidoc/entity-view/manual/en_US/10_type_converter.adoc [hunk 6]
```diff
@@ -0,0 +1,37 @@
+[[entity-view-type-converter-api]]
+== TypeConverter API
+
+The TypeConverter API is similar to the JPA AttributeConverter API as it allows to convert between an entity view model type and an underlying type.
+This it similar to the BasicUserType SPI but can also be used to convert view types to custom types.
```
Comment (Mobe91) [inline documentation/src/main/asciidoc/entity-view/manual/en_US/10_type_converter.adoc]: This is similar ...
Add link to BasicUserType SPI documentation?

**Comment by beikov** [comment]
a32d73edb3ac2ed95edb3f6407ea8b1d5e53a1bf

**Pull request closed without merging**